doc/getting-started/apl-nuc.rst

```diff
@@ -185,7 +185,7 @@ partition. Follow these steps:
 
    A starter acrn.conf configuration file is included in the Clear Linux release and is
    also available in the acrn-hypervisor/hypervisor GitHub repo as `acrn.conf
-   <https://github.com/projectacrn/acrn-hypervisor/hypervisor/tree/master/bsp/uefi/clearlinux/acrn.conf>`__
+   <https://github.com/projectacrn/acrn-hypervisor/blob/v0.1/hypervisor/bsp/uefi/clearlinux/acrn.conf>`_
```

Comment:
this tagged branch (v0.1) doesn't exist (yet).  Probably best to stick with the master branch as it was written.